@xen-orchestra/lite/src/App/TreeView.tsx

```diff
@@ -48,7 +50,7 @@ const TreeView = withState<State, Props, Effects, Computed, ParentState, ParentE
         if (state.pools === undefined) {
           return
         }
-        const collection = []
+        const collection: ItemType[] = []
         state.pools.valueSeq().forEach((pool: Pool) => {
           const haltedVms = []
```

Comment:
you can add explicitly a `any[]` type , to remove the warning

Reply:
yes, itemType is even better, can you also apply it to `haltedVms`, `hosts` and `runningVms`. That way, only the warning on the id of IntlMessage will stay. But I don't know how to resolve it